### DISCUSSION TITLE
Système de karma

### DISCUSSION THREAD

**Discussion by Pietagorh** [Crownicles/Crownicles#594] [Idées acceptées / déjà implémentées]
En fonction des choix des utilisateurs, calculer un score de karma qui aura une influence sur le joueur (changements des statistiques ou encore changement des résultats des évènements)

**Comment by Pietagorh** [comment]
Potentiellement une classe qui réduit/désactive les effets du karma

**Comment by Bombafox** [comment]
Cette suggestions je m'en rappelle quand y avait l'histoire de donner des pièces ou pas mdrr
C'est devenu un mythe
